.github/workflows/container-build.yml

```diff
@@ -26,11 +26,11 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
These changes look good, but if you can it would be nice if you could test it in your fork. I see that the workflow is disabled in your fork right now.

This is not a merge blocker.

Reply:
Updated the workflow and here's an example run (it will only push the image when building tags): https://github.com/nex-health/terragrunt-atlantis-config/actions/runs/7737813075/job/21097442628